Fix action issue - `Precommit - SN-Platform-Vault Helm Chart (Basic Template)`

https://github.com/streamnative/charts/runs/6999971834?check_suite_focus=true

```log
Error: can't get a valid version for repositories sn-platform. Try changing the version constraint in Chart.yaml
Error: Process completed with exit code 1.
```

--- a/.github/workflows/sn-platform-stack.yml
+++ b/.github/workflows/sn-platform-stack.yml
@@ -17,7 +17,7 @@
 # under the License.
 #
 
-name: Precommit - SN-Platform-Vault Helm Chart (Basic Template)
+name: Precommit - SN-Platform-Stack Helm Chart (Basic Template)
 on:
   pull_request:
     branches:

--- a/.github/workflows/sn-platform-vault.yml
+++ b/.github/workflows/sn-platform-vault.yml
@@ -17,7 +17,7 @@
 # under the License.
 #
 
-name: Precommit - SN-Platform-Stack Helm Chart (Basic Template)
+name: Precommit - SN-Platform-Vault Helm Chart (Basic Template)
 on:
   pull_request:
     branches:

--- a/charts/sn-platform-stack/Chart.yaml
+++ b/charts/sn-platform-stack/Chart.yaml
@@ -33,7 +33,7 @@ maintainers:
 dependencies: 
   - name: sn-platform
     version: "1.4.0"
-    repository: file://../sn-platform
+    repository: https://charts.streamnative.io
   - name: sn-console
     version: "1.9.0"
     repository: https://charts.streamnative.io

